### ISSUE TITLE
Capacity bar style is broken

### ISSUE THREAD

**Issue by Zzombiee2361** [Luwx/Lightly#69]
![image](https://user-images.githubusercontent.com/37852137/107867930-04e0af80-6eb2-11eb-8d10-f69c57556857.png)

I'm using latest master branch

**Comment by D3SOX** [comment]
Can't reproduce on my end.
![a](https://user-images.githubusercontent.com/24937357/107873600-fb584700-6eb3-11eb-944d-0164e62498d1.png)
Which Plasma Style and Colors are you using?

**Comment by Zzombiee2361** [comment]
I'm using breeze dark color scheme, the same thing happens with lightly color scheme too. Is there any way I could debug this?

**Comment by D3SOX** [comment]
Hm I'm not sure. Maybe it has something to do with your Plasma/KDE Frameworks version.
I'm running Plasma 5.20.9 with KDE Frameworks 5.78.0

**Comment by clwe** [comment]
![lightly_capacity_bar](https://user-images.githubusercontent.com/9591738/107877016-01592280-6eca-11eb-9704-108af7af1256.jpg)
I have a similar issue in 
Kubuntu 20.10
KDE Plasma Version: 5.19.5
KDE Frameworks Version: 5.74.0
Qt Version: 5.14.2

**Comment by clwe** [comment]
How do I update Plasma and Frameworks? I tried with as described here, from the backports repository. 
https://community.kde.org/Kubuntu/PPAs#Kubuntu_Backports
It still says up-to-date though. Do I have to build from source?

**Comment by Zzombiee2361** [comment]
Even after updating to KDE Frameworks to 5.79.0 and KDE Plasma to 5.21.0, then uninstalling and rebuilding Lightly, the issue persist.

**Comment by Vitorvlv** [comment]
Same issue
Plasma: 5.20.5
Qt: 5.15.2
KDE Frameworks: 5.78.0

![Screenshot_20210220_165241](https://user-images.githubusercontent.com/26132870/108606934-14716400-739c-11eb-8788-93e7999b1590.png)

**Comment by swiatlyk** [comment]
Same for me. Tested on KDE NEON with plasma 5.21, 5.20.5 and on KUBUNTU with Plasma 5.19.5

**Comment by Luwx** [comment]
I'll try to reproduce this in a KDE Neon vm. Meanwhile, the Kde people could really change the default look of the KCapacityBar to something more pleasant, it looks so out of place.

**Comment by RizzoV** [comment]
Can confirm the bug is present on:

Debian Bullseye/11 (sid)
KDE Plasma: 5.20.5
KDE Frameworks: 5.78.0
Qt: 5.15.2

![a](https://user-images.githubusercontent.com/12030659/110217061-60e79400-7eb2-11eb-9af2-281230043e23.png)

**Comment by chanleyuu** [comment]
It's affecting me too 
KDE Plasma 5.21.1
KDE Framework 5.79.0
QT 5.15.2
![capacity bars](https://user-images.githubusercontent.com/56023204/110781240-8fbc8c00-825d-11eb-9969-1880a44b5ab9.png)

**Comment by Mistsuu** [comment]
It also affects me.
KDE Plasma 5.21.4
KDE Framework Version 5.80.0
Qt Version 5.15.2
![image](https://user-images.githubusercontent.com/49508912/115963904-de8f5e00-a54b-11eb-9e54-98302e1ae062.png)

**Comment by SayanChakroborty** [comment]
Weirdly this issues is only present on openSUSE Tumbleweed. I have tested on Arch, Kubuntu, Neon, Fedora, KaOS... None of them had this issue. Only openSUSE Tumbleweed.

**Comment by SayanChakroborty** [comment]
> I'll try to reproduce this in a KDE Neon vm. Meanwhile, the Kde people could really change the default look of the KCapacityBar to something more pleasant, it looks so out of place.

Until the issue is fixed, can we as users modify something to make KCapacityBar not look out-of-place?

**Comment by deter0** [comment]
> Weirdly this issues is only present on openSUSE Tumbleweed. I have tested on Arch, Kubuntu, Neon, Fedora, KaOS... None of them had this issue. Only openSUSE Tumbleweed.

I have this issue on Manjaro:
![image](https://user-images.githubusercontent.com/82973108/148718873-51982b61-1ff2-46c0-9f34-34f64ab059aa.png)

**Comment by xuzhao9** [comment]
> openSUSE

Can you try if [this package](https://build.opensuse.org/package/show/home:nuklly:branches:KDE:Extra/lightly) fixes your problem?

**Comment by SayanChakroborty** [comment]
> > openSUSE
> 
> @SayanChakroborty Can you try if [this package](https://build.opensuse.org/package/show/home:nuklly:branches:KDE:Extra/lightly) fixes your problem?

I just tried the package you linked on openSUSE Tumbleweed snapshot 20220409 but the capacity bar is still broken.
![Screenshot_20220412_071302](https://user-images.githubusercontent.com/62809839/162948601-1f27025e-5177-42a7-a66f-5c1f0019e880.png)
![Screenshot_20220412_071426](https://user-images.githubusercontent.com/62809839/162948632-388e846d-0cd7-46ad-927b-da858ace0b7a.png)

**Comment by Voltstro** [comment]
I managed to fix this issue on two of my installs of Kubuntu 21.10.
You need to have KDE's [KF5FrameworkIntegration](https://invent.kde.org/frameworks/frameworkintegration) package installed (both the binary and dev packages).

On Ubuntu's repository they are called `libkf5style5` & `libkf5style-dev`. IDK about other distros.

This is happening because in the cmake file the KF5FrameworkIntegration package is marked as optional.

**Comment by xuzhao9** [comment]
> > > openSUSE
> > 
> > 
> > @SayanChakroborty Can you try if [this package](https://build.opensuse.org/package/show/home:nuklly:branches:KDE:Extra/lightly) fixes your problem?
> 
> I just tried the package you linked on openSUSE Tumbleweed snapshot 20220409 but the capacity bar is still broken. ![Screenshot_20220412_071302](https://user-images.githubusercontent.com/62809839/162948601-1f27025e-5177-42a7-a66f-5c1f0019e880.png) ![Screenshot_20220412_071426](https://user-images.githubusercontent.com/62809839/162948632-388e846d-0cd7-46ad-927b-da858ace0b7a.png)

Can you try install `frameworkintegration-devel` and `frameworkintegration-plugin` packages on Tumbleweed?

**Comment by SayanChakroborty** [comment]
> > > > openSUSE
> > > 
> > > 
> > > @SayanChakroborty Can you try if [this package](https://build.opensuse.org/package/show/home:nuklly:branches:KDE:Extra/lightly) fixes your problem?
> > 
> > 
> > I just tried the package you linked on openSUSE Tumbleweed snapshot 20220409 but the capacity bar is still broken. ![Screenshot_20220412_071302](https://user-images.githubusercontent.com/62809839/162948601-1f27025e-5177-42a7-a66f-5c1f0019e880.png) ![Screenshot_20220412_071426](https://user-images.githubusercontent.com/62809839/162948632-388e846d-0cd7-46ad-927b-da858ace0b7a.png)
> 
> Can you try install `frameworkintegration-devel` and `frameworkintegration-plugin` packages on Tumbleweed?

Yes, installing those packages solves the issue. Thank you so much.

**Comment by Achardir** [comment]
In case someone does not know, as mentioned by Voltstro, installing `libkf5style-dev` fixed this for me on Neon (22.04) and Kubuntu (23.04). Need to uninstall Lightly and then reinstall for it to work btw.
